Packs/CommunityCommonScripts/Scripts/FindPlaybookCustomDependencies/FindPlaybookCustomDependencies.py

```diff
@@ -0,0 +1,98 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+def get_custom_scripts_playbooks():
+    results = {}
+    final_results = []
+    req_response = demisto.executeCommand(
+        "demisto-api-post", {"uri": "automation/search", "body": "{\"query\":\"system:F AND hidden:F AND deprecated:F\"}"})
+    list_scripts = req_response[0].get('Contents', {}).get('response', {}).get('scripts')
+    if not list_scripts:
+        return_error("No custom scripts found.")
```

Comment:
If a search command doesn't return any results, return a human readable message and no Context data, do not return_error
```suggestion
        return_results(CommandResults(
            readable_output="No custom scripts found.",
        ))
```